tests/test_printer.py

```diff
@@ -357,17 +357,14 @@ class PlusCustomFormatOp(Operation):
     def parse(cls, result_types: List[Attribute],
               parser: BaseParser) -> PlusCustomFormatOp:
 
-        lhs = parser.expect(parser.try_parse_value_id,
-                            'Expected SSA Value name here!')
+        parser.parse_operand()
```

Comment:
This has to be wrong, right? :sweat_smile:

Reply:
Yes, this wasn't 100% ready ^^'